Lesson_03/src/main/java/sarapulova/task_01/Vector.java

```diff
@@ -0,0 +1,141 @@
+package sarapulova.task_01;
+
+public class Vector {
+    double vector_x;
+    double vector_y;
+    double vector_z;
+
+    Vector(double i, double j, double k) {
+        vector_x = i;
+        vector_y = j;
+        vector_z = k;
+    }
+
+    static double vectorLength(Vector vector)
+    {
+        double res;
+        res = Math.sqrt(Math.pow(vector.vector_x,2) + Math.pow(vector.vector_y,2) + Math.pow(vector.vector_z,2));
```

Comment:
Можно сразу написать вот так: 
```suggestion
        return Math.sqrt(Math.pow(vector.vector_x,2) + Math.pow(vector.vector_y,2) + Math.pow(vector.vector_z,2));
```
а создание res убрать